conf/webpack.conf.js

```diff
@@ -8,9 +8,16 @@ module.exports = {
             {
                 test: /\.js$/,
                 exclude: /node_modules/,
-                loaders: [
-                    'babel-loader?presets[]=es2015&presets[]=react&plugins[]=transform-object-assign'
-                ]
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        presets: ['es2015', 'react'],
+                        plugins: [
+                            'transform-object-assign',
+                            'transform-object-rest-spread'
+                        ]
+                    }
+                }
```

Comment:
If you're feeling charitable you could also make these changes to the boilerplate this project is based on https://github.com/guardian/editorial-tools-play-template